code/modules/assembly/holder.dm

```diff
@@ -149,9 +156,9 @@
 /obj/item/assembly_holder/proc/process_activation(obj/device, normal = TRUE, special = TRUE)
 	if(!device)
 		return FALSE
-	if(normal && LAZYLEN(assemblies) >= 2)
+	if(normal && assemblies.len >= 2)
```

Comment:
Listen, it's your codebase, but I will tell you that it gives me absolute hives to see a mystery macro, especially when it equates to a builtin function.  As a programmer it gives me _negative_ trust in the code to see things like this, especially when the LAZYACCESS macro elsewhere was the direct cause of an error.  It is _genuinely_ hard for me to believe that a mystery macro is more resilient than a builtin var, but I will certainly yield if you request that this is reverted.

Reply:
that's nice

Reply:
Having had this explained to me, I will say it will be reverted.  To be clear: I did look up the macro to see what it did, and the fact that it is there and not explained at all is the whole reason why it's creepy.